app/frontend/src/features/messages/useSendFieldStyles.ts

```diff
@@ -3,13 +3,14 @@ import { makeStyles } from "@material-ui/core";
 const useSendFieldStyles = makeStyles((theme) => ({
   button: {
     [theme.breakpoints.up("md")]: {
-      margin: 0,
       marginInlineStart: theme.spacing(2),
+      marginBottom: theme.spacing(1),
       marginTop: theme.spacing(1),
     },
     display: "block",
     flexShrink: 0,
     marginInline: "auto",
```

Comment:
I know I wrote this (lol) but can you change this line to `marginInlineStart` and `marginInlineEnd` since this shorthand is not well supported yet.